src/os/bluestore/bluestore_types.h

```diff
@@ -111,7 +111,7 @@ WRITE_CLASS_DENC(bluestore_pextent_t)
 
 std::ostream& operator<<(std::ostream& out, const bluestore_pextent_t& o);
 
-typedef mempool::bluestore_cache_other::vector<bluestore_pextent_t> PExtentVector;
+typedef mempool::bluestore_extent::vector<bluestore_pextent_t> PExtentVector;
```

Comment:
I would suggest to attach this to bluestore_blob mempool, not bluestore_extent one. The latter is intended primarily for logical extents while physical extents are rather part of blob infra.